app/src/main/java/com/sun/note_01/data/source/UserRepository.kt

```diff
@@ -0,0 +1,39 @@
+package com.sun.note_01.data.source
+
+import com.sun.note_01.data.source.local.UserLocalDataSource
+import com.sun.note_01.data.source.remote.OnSendDataJsonListener
+import com.sun.note_01.data.source.remote.UserRemoteDataSource
+
+class UserRepository private constructor(
+    private val remote: UserDataSource.Remote,
+    private val local: UserDataSource.Local
+) :
+    UserDataSource.Local, UserDataSource.Remote {
```

Comment:
chú ý header của class 
```suggestion
class UserRepository private constructor(
    private val remote: UserDataSource.Remote,
    private val local: UserDataSource.Local
) : UserDataSource.Local, UserDataSource.Remote {
```